packages/react-composites/package.json

```diff
@@ -16,7 +16,7 @@
     "build:e2e:chat": "webpack-cli build --config ./tests/browser/chat/app/webpack.config.js --mode=production --env production",
     "test": "jest",
     "test:coverage": "npm run test -- --coverage",
-    "test:e2e:chat": "playwright test chat",
+    "test:e2e:chat": "playwright test tests/browser/chat",
```

Comment:
This won't work. Could you double check?

Reply:
Work for me. Why should this not work?

The positional argument is treated as regexp, so `chat` is a bit too broad (it could easily pick up jest tests from `src/*`)
![image](https://user-images.githubusercontent.com/82062616/125992405-25f9acb4-1bc7-4041-ae91-e86afdfc855e.png)

Reply:
Try running the command directly.
`npx playwright test tests/browser/chat`
Does it work?

Reply:
Doesn't work for me. Just checked.

Reply:
![image](https://user-images.githubusercontent.com/9782747/125994403-6bde4475-43f2-458d-ba00-3eeaa202b034.png)

Reply:
I think this is because you are running it in codespaces on a linux environment.
Windows and Linux have different path resolution

Reply:
You can verify by running it on your dev machine using this
`npx playwright test tests\\browser\\chat`

Reply:
Great catch. Verified that using `.` works for both, as this is a regexp.